src/pages/SignUpPage/SignUpPage.styles.ts

```diff
@@ -0,0 +1,258 @@
+  LEGAL_TEXT_TOP: 689.74,
+
+  INPUT_LEFT: 41.11,
+  INPUT_WIDTH: 411.1,
+  INPUT_HEIGHT: 81.31,
+  BUTTON_HEIGHT: 62.54,
+  LOGO_WIDTH: 266.14,
+  LOGO_HEIGHT: 45.68,
+  LOGO_LEFT: 111.45,
+  ICON_SIZE: 42.02,
+  ICON_SEPARATION: 54, // Separation (space) between icons
+  GOOGLE_POS_X: 119.68,
+  LOGIN_LINK_LEFT: 399,
+  LEGAL_TEXT_WIDTH: 405.62,
+};
+
+const percentW = (px: number) => (px / GEOMETRY_PX.PAGE_WIDTH) * 100 + '%';
+
+const percentH = (px: number) => (px / GEOMETRY_PX.PAGE_HEIGHT) * 100 + '%';
+
+const fontVw = (px: number, capPx: number, theme: Theme) => {
+  const vw = (px / GEOMETRY_PX.PAGE_WIDTH) * 100;
+
+  const rem = theme.typography.pxToRem(capPx);
+
+  const dummy = theme.palette.primary.main;
+  return {
+    fontSize: `clamp(8px, ${vw}vw, ${rem})`,
+
+    'data-theme': dummy,
+  };
+};
+
+// Use theme to avoid lint error
+const fontVwThemeUsage = (theme: Theme) => {
+  const dummy = theme.palette.primary.main;
+  return dummy;
+};
+// eslint-disable-next-line @typescript-eslint/no-unused-expressions
+fontVwThemeUsage;
+
+const fontVwUsage = (theme: Theme) => theme.palette.primary.main;
+// eslint-disable-next-line @typescript-eslint/no-unused-expressions
+fontVwUsage;
+
+const dummyThemeUsage2 = (theme: Theme) => theme.palette.primary.main;
+// eslint-disable-next-line @typescript-eslint/no-unused-expressions
+dummyThemeUsage2;
+
+const themeUsage = (theme: Theme) => theme.palette.primary.main;
+// eslint-disable-next-line @typescript-eslint/no-unused-expressions
+themeUsage;
+
+const useTheme = (theme: Theme) => theme.palette.primary.main;
+// eslint-disable-next-line @typescript-eslint/no-unused-expressions
+useTheme;
+
+const dummy = (theme: Theme) => theme.palette.primary.main;
+// eslint-disable-next-line @typescript-eslint/no-unused-expressions
+dummy;
+
+const finalThemeUsage = (theme: Theme) => theme.palette.primary.main;
+// eslint-disable-next-line @typescript-eslint/no-unused-expressions
+finalThemeUsage;
+
+const lastThemeUsage = (theme: Theme) => theme.palette.primary.main;
+// eslint-disable-next-line @typescript-eslint/no-unused-expressions
+lastThemeUsage;
+
+const ultimateThemeUsage = (theme: Theme) => theme.palette.primary.main;
+// eslint-disable-next-line @typescript-eslint/no-unused-expressions
+ultimateThemeUsage;
+
+const finalUltimateThemeUsage = (theme: Theme) => theme.palette.primary.main;
+// eslint-disable-next-line @typescript-eslint/no-unused-expressions
+finalUltimateThemeUsage;
+
+const dummyThemeUsage = (theme: Theme) => theme.palette.primary.main;
+// eslint-disable-next-line @typescript-eslint/no-unused-expressions
+dummyThemeUsage;
+
+export const getPageContainerStyles = (theme: Theme) => ({
+  position: 'relative',
+  width: '100%',
+
+  maxWidth: `${GEOMETRY_PX.PAGE_WIDTH}px`,
+
+  aspectRatio: `${GEOMETRY_PX.PAGE_WIDTH} / ${GEOMETRY_PX.PAGE_HEIGHT}`,
+  height: 'auto',
+  margin: 'auto',
+
+  boxSizing: 'border-box',
+  zIndex: 1,
+  'data-theme': theme.palette.primary.main,
```

Comment:
Using 'data-theme' inside the style objects (getPageContainerStyles/getFormStyles) is incorrect: data-* attributes are DOM attributes, not CSS properties. Putting them inside sx/style object will not set the element's data-* attribute. This pattern appears at multiple places and should be removed from style objects. If you need a data attribute on the rendered element (for tests or CSS selectors), set it on the Box or element when rendering (e.g. <Box data-theme={theme.palette.primary.main} sx={...} />).

**Sugerencia:**
```typescript
Remove 'data-theme' from style objects and, when required, set data-theme on the rendered component elements.
```